airflow/datasets/__init__.py

```diff
@@ -179,6 +183,34 @@ def __eq__(self, other: Any) -> bool:
     def __hash__(self) -> int:
         return hash(self.name)
 
+    def as_expression(self) -> Any:
+        """
+        Serialize the dataset into its scheduling expression.
+
+        :meta private:
+        """
+        return {"any": [o.as_expression() for o in self.expand_datasets()]}
```

Comment:
I wonder if this should just be `{"alias": self.name}`. This is only used for the UI, and I kind of suspect people want to know the alias name more than what datasets it’s expanded into.

Reply:
I thought it's used for scheduling as well 🤔 If that's the case, yes `{"alias": self.name}` makes more sense

Reply:
updated